docs/paper/paper.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Qiskit Experiments: A Python package to characterize and calibrate quantum computers'
+tags:
+  - Python
+  - Quantum computing
+  - Characterization
+  - Calibration
+authors:
+  - name: Naoki Kanazawa
+    orcid: 0000-0002-4192-5558
+    affiliation: 1
+  - name: Daniel J. Egger
+    orcid: 0000-0002-5523-9807
+    corresponding: true
+    affiliation: 2
+  - name: Yael Ben-Haim
+    orcid: 0000-0000-0000-0000
+    affiliation: 3
+  - name: Helena Zhang
+    orcid: 0000-0002-7813-7133
+    affiliation: 4
+  - name: Will E. Shanks
+    orcid: 0000-0002-5045-8808
+    affiliation: 4
+  - name: Gadi Aleksandrowicz
+    orcid: 0000-0000-0000-0000
+    affiliation: 3
+  - name: Christopher J. Wood
+    orcid: 0000-0001-7606-7349
+    affiliation: 4
+affiliations:
+ - name: IBM Quantum – IBM Research Tokyo, Tokyo, 103-8510, Japan
+   index: 1
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon 8003, Switzerland
+   index: 2
+ - name: IBM Quantum, IBM Research Israel, Haifa 31905, Israel
+   index: 3
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY 10598, USA
+   index: 4
+date: 21 February 2023
```

Comment:
Should the date be updated before merging?

Reply:
```suggestion
date: 14 March 2023
```

Reply:
Done here: [7cc3d20](https://github.com/Qiskit/qiskit-experiments/pull/1055/commits/7cc3d20c85476b161be3864f7a015d73934c569a)